playwright/fixtures/makeName/generators/generateLegacyName.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * NOTE: This function simulates the method used by ENSVision to generate names. It is not used in a test yet so it might
+ * have some bugs that need to be fixed.
+ */
```

Comment:
ENSVision uses the method commit/register instead of commitWithConfig/registerWithConfig. It let's them register names without a resolver and lower gas costs